README.md

```diff
@@ -17,8 +17,14 @@ First you need to [install V](https://github.com/vlang/v#installing-v-from-sourc
 git clone https://github.com/vlang/vid
 cd vid
 v .
+```
```

Comment:
why close and reopen the code block, I think there should be more clarity (for less confusion)

Reply:
> why close and reopen the code block, I think there should be more clarity (for less confusion)

It should be clear that the "on Windows" should only be applied to the last line of commands

Reply:
That's what I was saying